src/utils/index.js

```diff
@@ -0,0 +1,53 @@
+
+export const mapComment = commentLen => {
```

Comment:
A few notes about this style of utility functions:

* When functions appear out of context, it is especially important to have comments describing them. Imagine a scenario where you can't see the implementation of a function, only its name and comments. `mapComment` would be pretty confusing in that case.

* Utility functions are also great candidates for unit tests.